<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDrawing</class>
 <widget class="QWidget" name="PlayerDrawing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QLabel" name="playerName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Player Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hitButton">
   <property name="geometry">
    <rect>
     <x>297</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Hit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="foldButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Fold</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerTimer">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="whatsThis">
    <UInt>0</UInt>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Timer:20 sec</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
